derive_client/constants.py

```diff
@@ -115,7 +115,7 @@ class EnvConfig(BaseModel, frozen=True):
 GAS_FEE_BUFFER = 1.1  # buffer multiplier to pad maxFeePerGas
 GAS_LIMIT_BUFFER = 1.1  # buffer multiplier to pad gas limit
 MSG_GAS_LIMIT = 200_000
-DEPOSIT_GAS_LIMIT = 420_000
+ASSUMED_BRIDGE_GAS_LIMIT = 1_000_000
```

Comment:
[because 600k+ is probable](etherscan.io/address/0x61e44dc0dae6888b5a301887732217d5725b0bff)

and I don't know of any better was to get a gas estimate that won't trigger JSON RPC errors when building or estimating gas